test/spec/modules/spotxBidAdapter_spec.js

```diff
@@ -564,6 +565,7 @@ describe('the spotx adapter', function () {
       expect(scriptTag.getAttribute('data-spotx_content_width')).to.equal('400');
       expect(scriptTag.getAttribute('data-spotx_content_height')).to.equal('300');
       expect(scriptTag.getAttribute('data-spotx_ad_mute')).to.equal('1');
+      expect(scriptTag.getAttribute('data-spotx_custom_container_styles')).to.equal('postion: absolute;');
```

Comment:
the first `_` here is a `-` above.  Looks like this is causing tests to fail.

Reply:
I would like to abandon this pull request.